tests/test_channel_creation.py

```diff
@@ -20,6 +20,7 @@
     StrainGageBridgeType,
     BridgeConfiguration,
 )
+from nidaqmx.tests.fixtures import sim_ts_power_device, any_x_series_device  # noqa: F401
```

Comment:
Just remove whatever isn't used

Reply:
pytest looks these up at run time, but flake8 doesn't know that.

I think the solution is to rename `fixtures.py` to `conftest.py` and remove the imports:
https://docs.pytest.org/en/4.6.x/fixture.html#conftest-py-sharing-fixture-functions

Reply:
(or import them into conftest.py)